util/log/file.go

```diff
@@ -221,3 +297,111 @@ func GetLogReader(filename string, allowAbsolute bool) (io.ReadCloser, error) {
 	}
 	return nil, err
 }
+
+// int64sortable is required so we can sort on an int64 slice.
+type int64sortable []int64
```

Comment:
Brilliant idea.  Reduced the size of the function and it's one lass pass through the list of files.